packages/account/src/Components/poi/status/unsupported/unsupported.tsx

```diff
@@ -34,11 +42,11 @@ const Unsupported = ({ country_code, handlePOIforMT5Complete, ...props }) => {
         return (
             <DetailComponent
                 is_onfido_supported={country_code === 'ng' && !checkNimcStep(documents[detail].details.documents)}
-                country_code={country_code}
+                country_code_key={country_code}
```

Comment:
I guess, it didn’t work before , as the params in props and here are different, could you please check if the functionality is as expected

Reply:
I've checked it works fine